addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/ContentSecurityPolicyMissingScanRule.java

```diff
@@ -146,4 +121,33 @@ private String getAlertAttribute(String key) {
     public Map<String, String> getAlertTags() {
         return ALERT_TAGS;
     }
+
+    private boolean hasCspHeader(HttpMessage msg) {
+        // Content-Security-Policy is supported by Chrome 25+, Firefox 23+, Safari 7+, but not but
+        // Internet Exploder
+        List<String> cspOptions =
+                msg.getResponseHeader().getHeaderValues(HttpFieldsNames.CONTENT_SECURITY_POLICY);
+
+        return !cspOptions.isEmpty();
```

Comment:
Make this a single statement, the extra variable is not needed pretty clear what's being checked.

Reply:
when I tried this, spotless forces the structure to be broken across 3 lines.  I find that quite a lot less readable -- it's not terribly clear that the exclamation mark is applying to the .isEmpty() that falls 2 lines below it.  I think it's important to keep the '!' and the condition that it is negating closer together than your suggestion allows.

Reply:
The negation does not apply to intermediate calls.

Reply:
I know, but with your suggestion, the code ends up looking like this:

```
    return !msg.getResponseHeader()
        .getHeaderValues(HttpFieldsNames.CONTENT_SECURITY_POLICY)
        .isEmpty();
```  
which obscures what exactly is being negated.